[MRG + 1] DOC

GaelVaroquaux opened the pull request:
Slightly more didactic docs

bthirion commented on doc/decoding/decoding_intro.rst:
```diff
@@ -64,36 +73,61 @@ corresponding category.
-Loading the data into Python
-----------------------------
+   The standard approach to decoding consists in fitting a first-level
+   GLM to retrieve one responce map (a beta map) per trial. This is
```
response

bthirion commented on doc/decoding/decoding_intro.rst:
```diff
@@ -64,36 +73,61 @@ corresponding category.
-  files downloaded on the disk.
+* **Retrieving the data**: In the tutorial, we load the data using nilearn
+  data downloading function, :func:`nilearn.datasets.fetch_haxby`.
+  However, all this function does is downloading the data and return 
```
'downloading and returning' or 'download and return'

bthirion commented on doc/decoding/decoding_intro.rst:
```diff
@@ -64,36 +73,61 @@ corresponding category.
   (see :ref:`mask_4d_2_3d` for a discussion on using masks).
 
-* **Sample mask**: Masking some of the data points may be useful to
+* **Sample mask**: Masking some of the times points may be useful to
```
time

bthirion commented:
LGTM.

GaelVaroquaux commented:
@bthirion : I've addressed your comments.

I've finished making the changes to the documentation that came to my head during the Stockholm tutorial. This is ready to merge.

KamalakerDadi commented on doc/decoding/decoding_intro.rst:
```diff
@@ -247,7 +286,8 @@ to use the session label, present in the behavioral data file, and
     0.76851...
 
-We have a total prediction accuracy of *77%* across the different sessions.
+For disriminating human faces from cats, we measure a total prediction
```
disriminating -> discr

KamalakerDadi commented on doc/decoding/decoding_intro.rst:
```diff
@@ -257,9 +297,10 @@ the number of total errors. It is not always a sensible metric,
 choosing the dominant class can achieve a low number of errors.
 
-Other metrics, such as the f1-score, can be used::
+Other metrics, such as the AUC (Aera Under the Curve, for the ROC: the
```
Aera -> Area

KamalakerDadi commented on doc/decoding/decoding_intro.rst:
```diff
@@ -257,9 +297,10 @@ the number of total errors. It is not always a sensible metric,
 
-Other metrics, such as the f1-score, can be used::
+Other metrics, such as the AUC (Aera Under the Curve, for the ROC: the
+Reciever Operating Characteristic), can be used::
```
Reciever -> Reciever

KamalakerDadi commented on doc/decoding/estimator_choice.rst:
```diff
@@ -209,7 +209,22 @@ little guarantee on the brain maps.
+
+.. topic:: **Decoding on simulated data**
+
+   Simple simulations may be useful to understood the behavior of a given
```
understood -> understand

KamalakerDadi commented on doc/decoding/estimator_choice.rst:
```diff
@@ -209,7 +209,22 @@ little guarantee on the brain maps.
+   :ref:`sphx_glr_auto_examples_02_decoding_plot_simulated_data.py`
+   Simulated data cannot easily mimic all properties of brain data. An
+   important aspect, however, is its spatial structure, that we create in
+   the similations.
```
similations -> simulations

KamalakerDadi commented on examples/02_decoding/plot_haxby_anova_svm.py:
```diff
@@ -21,32 +21,34 @@
-# Restrict to faces and houses
-condition_mask = np.logical_or(y == b'face', y == b'house')
-y = y[condition_mask]
+# Restrict the analysis to faces and places
```
places -> animals

KamalakerDadi commented on examples/02_decoding/plot_haxby_anova_svm.py:
```diff
@@ -71,55 +73,52 @@
 
 
-# Create the figure
+# USe the mean image as a background to avoid relying on anatomic data
```
USe -> Use
anatomic -> anatomical

GaelVaroquaux commented:
Thanks for the comments. I addressed all of them.

KamalakerDadi commented:
+1 for merge.

GaelVaroquaux commented:
Thanks for the review!
